zigzag/module_hierarchy_facade.py

```diff
@@ -20,7 +20,6 @@ def __init__(self, mediator):
 
         self._mediator = mediator
         self._pr_regex = re.compile(r'.*(PR-\d+).*')
```

Comment:
If `_pr_match` is gone, is this still useful?

Reply:
Nope its not. @ryan-rs Just pushed a commit to remove it.